core/src/main/java/bio/terra/pearl/core/service/workflow/EnrollmentService.java

```diff
@@ -220,6 +220,7 @@ private void backfillPreEnrollResponse(PortalParticipantUser operator, Enrollee
                         .builder()
                         .surveyId(preEnrollResponse.getSurveyId())
                         .answers(answers)
+                        .responseMetadata(preEnrollResponse.getReferralSource())
```

Comment:
will we want (maybe in a future PR) to rename "getReferralSource()" to "getResponseMetadata" since that's how we use it?